src/main/resources/hibernate.cfg.xml

```diff
@@ -0,0 +1,12 @@
+<hibernate-configuration>
+    <session-factory>
+        <property name="hibernate.dialect">org.hibernate.dialect.MySQL5Dialect</property>
+        <property name="connection.url">jdbc:mysql://localhost:3306/hibernate_configuration_db?serverTimezone=GMT</property>
+        <property name="connection.driver_class">com.mysql.cj.jdbc.Driver</property>
+        <property name="connection.username">root</property>
+        <property name="connection.password">entEr-41S</property>
+        <property name="show_sql">true</property>
+        <property name="hbm2ddl.auto">create-drop</property>
+        <mapping class="mate.academy.model.Movie"/>
```

Comment:
add a line between line 9 and 10 for better readability

Reply:
Thanks a lot!